kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -0,0 +1,309 @@
+<template>
+
+  <UiToolbar>
+    <KLabeledIcon :style="{ 'margin-top': '8px' }">
+      <template #icon>
+        <LearningActivityIcon
+          v-if="learningActivityKind"
+          data-test="learningActivityIcon"
+          :kind="learningActivityKind"
+          :shaded="true"
+        />
+      </template>
+      <TextTruncator
+        :text="resourceTitle"
+        :maxHeight="26"
+      />
+    </KLabeledIcon>
+
+    <template #icon>
+      <KIconButton
```

Comment:
I think it may be a good idea to add `ariaLabel` and `tooltip` for the back navigation button but am not sure yet and don't know what the string would look like. See https://learningequality.slack.com/archives/C023DNC5GCX/p1623850266060400.

Reply:
Also, it may need to be wrapped in `router-link` in the future depending on how we'll use the component. I took the simplest way for now thinking that we can update it together with a new prop for the link destination, instead of emitting `navigateBack` event if needed.